beps/bep_0007.rst

```diff
@@ -23,32 +24,31 @@ be interpreted as described in IETF `RFC 2119`_.
 
 .. _`RFC 2119`: http://tools.ietf.org/html/rfc2119
 
-Announce Parameter
-==================
+Announcing
+==========
+
+For each local IP address:
 
-The client MAY add an ``&ipv6=`` parameter to the HTTP GET request it sends
-to the tracker. The value is either an IPv6 endpoint (address and port) or
-just an IPv6 address. In the case where only an address is supplied, the IPv6
-port is assumed to be the same as specified by the ``&port=`` parameter.
+	1. the client intends to receive incoming peer connections over (listens on)
+	2. the client intends to publish to the tracker
```

Comment:
I think this is the key part. Clients need to consider which addresses they publish. This is tricky in multi-homed environments (hence several paragraphs on this topic in [BEP45](http://bittorrent.org/beps/bep_0045.html#recommended-optional)).

Perhaps we should state that the most simple case is a listening socket bound to the wild card address, which then does not require binding of the announce (modulo address families) and that more complex logic is only needed when custom binding or multihoming is in play.